scripts/program_analysis/cast_to_agraph.py

```diff
@@ -0,0 +1,77 @@
+import sys
+
+from automates.program_analysis.CAST2GrFN.visitors.cast_to_agraph_visitor import CASTToAGraphVisitor
+from automates.program_analysis.CAST2GrFN.cast import CAST
+from automates.program_analysis.CAST2GrFN.model.cast import (
+    AstNode,
+    Assignment,
+    Attribute,
+    BinaryOp,
+    BinaryOperator,
+    Call,
+    ClassDef,
+    Dict,
+    Expr,
+    FunctionDef,
+    List,
+    Loop,
+    ModelBreak,
+    ModelContinue,
+    ModelIf,
+    ModelReturn,
+    Module,
+    Name,
+    Number,
+    Set,
+    String,
+    Subscript,
+    Tuple,
+    UnaryOp,
+    UnaryOperator,
+    VarType,
+    Var,
+)
+
+def basic_function_def_and_assignment_cast():
+    v = Var(val=Name(name="exampleVar"), type="string")
+    n = String(string="hey")
+    a = Assignment(left=v, right=n)
+    f = FunctionDef(name=Name(name="exampleFunction"), func_args=[], body=[a])
+    m = Module(name="ExampleModule", body=[f])
+    return CAST([m])
+
+def basic_function_def_and_assignment_math():
+    v = Var(val=Name(name="exampleVar"), type="float")
+    n1 = Number(number=36.2)
+    n2 = Number(number=37.8)
+    op = BinaryOp(op="Add",left=n1,right=n2)
+    a = Assignment(left=v, right=op)
+    f = FunctionDef(name=Name(name="exampleFunction"), func_args=[], body=[a])
+    m = Module(name="ExampleModule", body=[f])
+    return CAST([m])
+
+def basic_function_def_and_assignment_if():
```

Comment:
I think you can remove these three functions that create CAST objects because you have updated main to read in a json file. You could also remove all the CAST imports if you remove these functions.

Reply:
Done!